support_levels.yml

```diff
@@ -0,0 +1,24 @@
+- name: Test support_levels
+  hosts: localhost
+  tasks:
+    - name: Query OpenShift architectures
+      support_levels:
+        resource_type: architectures
```

Comment:
curious if this would work if this was just `architectures:`?

Reply:
It could be made to work, but I think passing a module argument with no value is ambiguous.

Reply:
Agreed.
The unambiguous and having a valid yaml would be to use `architectures: {}` i.e an empty dictionary

Reply:
I realized that `/v2/component-versions` and `/v2/release-sources` have no parameters and will need a clean way to retrieve information when using `component-versions` and `release-sources` as module names in a play book.

Reply:
Sure, that is fine. A module that doesn't require or accept parameters is normal. My concern is with passing empty module parameters which, in my opinion, does not clearly specify intent.